docs/hello_nextflow/09_hello_nf-core.md

```diff
@@ -244,4 +244,1024 @@ You have a general idea of what nf-core offers and you know how to run an nf-cor
+-[myorg/myfirstpipeline] Pipeline completed successfully-
+```
+
+Let's dissect what we are seeing:
+
+The nf-core pipeline template is a working pipeline and comes pre-configured with some modules, here we only chose MultiQC:
+
+-   [MultiQC](https://multiqc.info/): A modular tool to aggregate results from bioinformatics analyses across many samples into a single report.
+
+!!! note "The template can be granularly configured"
+
+    From nf-core tools 3.0 onwards many features can be removed during template creation. This is what we did when deselecting 
+    features earlier.
+
+At the top you see, all parameters displayed that differ from the pipeline defaults. These were configured with the `test` profile.
+
+You can use the `test` profile to check if your pipeline is still working during your development cycle. 
+
+The default template `test` profile leverages small test files that are stored in the nf-core [test data GitHub repository](https://github.com/nf-core/test-datasets) as inputs for the pipeline.
+
+Additionally, the template comes with profiles for the management of software dependencies (e.g., `docker`, `singularity`, and `conda`). nf-core modules come with containers/images/recipes and profiles can be used to change the way dependencies are handled when you execute your pipeline.
+
+!!! warning
+
+    If `-profile` for managing software dependencies is not specified, the pipeline will run locally and expect all software to be installed and available on `PATH`. **This is not recommended.**
+
+Additional test profiles can be created to test different parts of your pipeline.
+
+### Template tour
+
+The nf-core pipeline template comes packed with a lot of files and folders.
+
+Here, we selected a subset (what we did in step 5 during the template creation).
+
+While the template may feel overwhelming, a complete understanding isn't required to start developing your pipeline. Let's look at the important places that we need to touch during pipeline development.
+
+#### Workflows, subworkflows, and modules
+
+The nf-core pipeline template has a `main.nf` script that calls `myfirstpipeline.nf` from the `workflows` folder. The `myfirstpipeline.nf` file inside the workflows folder is the central pipeline file that is used to bring everything else together.
+
+Instead of having one large monolithic pipeline script, it's broken up into smaller script components, namely, modules and subworkflows:
+
+-   **Modules:** Wrappers around a single process
+-   **Subworkflows:** Two or more modules that are packaged together as a mini workflow
+
+<figure class="excalidraw">
+--8<-- "docs/nf_develop/img/nested.excalidraw.svg"
+</figure>
+
+Within your pipeline repository, `modules` and `subworkflows` are stored within `local` and `nf-core` folders. The `nf-core` folder is for components that have come from the nf-core GitHub repository while the `local` folder is for components that have been developed independently:
```

Comment:
```suggestion
Within your pipeline repository, `modules` and `subworkflows` are stored within `local` and `nf-core` folders. The `nf-core` folder is for components that have come from the nf-core GitHub repository while the `local` folder is for components that have been developed independently (usually things very specific to a pipeline):
```

Maybe?

Reply:
let's go with this for now. In a new iteration it could be maybe, install module, patch module, local module to make that even more obvious. But for that I think the training has to be broken up more.